packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -149,9 +149,26 @@ export interface AwsCustomResourceProps {
   /**
    * The timeout for the Lambda function implementing this custom resource.
    *
-   * @default Duration.seconds(30)
+   * @default Duration.seconds(30) when `useLatestSdk` is false,
+   * Duration.seconds(60) otherwise.
    */
   readonly timeout?: cdk.Duration
+
+  /**
+   * Install and use the latest version of AWS SDK JS.
+   *
+   * When set to `true`, the latest v2 of AWS SDK JS will be installed when a
+   * new container is initialized for the Lambda function implementing this
+   * custom resource. Subsequent executions reusing this container will skip
+   * installation.
+   *
+   * As this custom resource uses a singleton Lambda function, it's important
+   * to note that this will apply to **all** `AwsCustomResource`s in the stack
+   * even if only one instance sets this parameters to `true`.
+   *
+   * @default false
```

Comment:
Wondering if this should be the default, because why not?

Reply:
Agree. If this is the default then the prop should be removed because to disable it it must be set to `false` on all `AwsCusomResource`?

Reply:
You mean from backwards compatibility perspective?

Reply:
I mean, do we want to offer a way to opt out from this feature because it will not be possible to opt out for some instances of `AwsCustomResource` only.

Reply:
Let's just make this the only behavior (remove the option to opt-out). I don't see value in using an old SDK...

Reply:
Done, added a fallback to the normal sdk (e.g. NPM unavailable).

Does this need documentation? Do you want to add this to `AwsApi` in `@aws-cdk/aws-events-targets`? In this PR?